Makefile

```diff
@@ -6,3 +6,6 @@ test:
 
 clean:
 	rm rancher-ecr-credentials
+
+push:
```

Comment:
This is fine for now, but we should move this to Dapper and all the other stuff we do in Go-Skel.